nixos/modules/programs/steam.nix

```diff
@@ -44,8 +44,8 @@ in {
       '';
       apply = steam: steam.override (prev: {
         extraEnv = (lib.optionalAttrs (cfg.extraCompatPackages != [ ]) {
-            STEAM_EXTRA_COMPAT_TOOLS_PATHS = makeBinPath cfg.extraCompatPackages;
-          }) // (prev.extraEnv or {});
+          STEAM_EXTRA_COMPAT_TOOLS_PATHS = makeSearchPathOutput "steamcompattool" "" cfg.extraCompatPackages;
```

Comment:
why don't we also add bin here?

Reply:
Non-executable files such as compat tool description files (vdf) do not belong in `bin/`.

Reply:
So we're planning to make sure every executable compat tool (i.e stl) provide an additional output?

Reply:
Yup; an output that only concerns the compat tool files. It wouldn't really be "additional" in most cases though as most compat tools one and only purpose is to act as a compat tool and would therefore have an empty `out` just like what I did to proton-ge-bin.